public/components/visualizations/plotly/plot.tsx

```diff
@@ -15,6 +15,8 @@ interface PltProps {
   onHoverHandler?: (event: Readonly<Plotly.PlotMouseEvent>) => void;
   onUnhoverHandler?: (event: Readonly<Plotly.PlotMouseEvent>) => void;
   onClickHandler?: (event: Readonly<Plotly.PlotMouseEvent>) => void;
+  onSelectedHandler?: (event: Readonly<Plotly.PlotSelectionEvent>) => void;
+  onRelayout?: (event: any) => void;
```

Comment:
Should this be `onRelayout?: (event: Readonly<Plotly.PlotSelectionEvent>) => void` can refrin from using any as type.

Reply:
Updated